### PR TITLE
Track command completions.

### PR DESCRIPTION
**Pull request by jchristgit**

### CODE REVIEW

File: bot/cogs/metrics.py [hunk 1]
```diff
@@ -71,6 +76,27 @@ def __init__(self, bot: Bot) -> None:
             guild_id=message.guild.id,
         ).inc()
 
+    @Cog.listener()
+    async def on_command_completion(self, ctx: Context) -> None:
+        """Increment the command completion counter."""
+        if ctx.message.guild is not None:
+            path = []
+            if (
```
Comment (jos-b) [inline bot/cogs/metrics.py]: This doesn't handle enough cases, why not use [this](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Command.parents)?

Comment (jos-b) [reply bot/cogs/metrics.py]: Cases I mean are like `!a b c` or deeper.

**Comment by jos-b** [comment]
We are both buffoons https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Command.full_parent_name

File: bot/cogs/metrics.py (hunk 1 above)
Comment (jchristgit) [reply bot/cogs/metrics.py]: b0b9cdd

**Pull request merged**